### DISCUSSION TITLE
월드 상품 제거하는 방법이 있나요?

### DISCUSSION THREAD

**Discussion by zlmrzlo** [naverz/zepeto-studio-kor#1157] [5. Q&A]
월드 출시 후에 월드 IWP를 적용하기 위해서 테스트를 하는 과정에서 월드 상품을 등록한 것이 있는데
이 부분을 제거하고 월드를 업데이트하고 싶은데 제거할 수 있는 방법이 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요,  현재는 IWP 상품 제거 기능은 스튜디오에서 제공하고 있지 않습니다.
새로운 버전 빌드시 말씀하신 IWP 상품을 사용하지 않도록 처리하신 후 심사 요청 주시면 됩니다 :) 
감사합니다.
